repository/reminder.repository.js

```diff
@@ -0,0 +1,47 @@
+import { Reminder } from '../models/index.js'
+import AppError from '../utils/appError.js'
+
+// Create reminder repository logic
+export const createReminder = (data) =>
+  Reminder.create(data)
+    .then((reminder) => {
+      return Promise.resolve(reminder)
+    }).catch((error) => {
+      throw new AppError(error)
```

Comment:
Instead of throwing the error, return the error. So that frontend API will know what went wrong without seeing the logs in the backend @dulshan-perera